docs/system_setup.rst

```diff
@@ -0,0 +1,390 @@
+8.  `Download example parameter sets <#download-example-parameter-sets>`__
+9.  `Prepare other parameter sets <#prepare-other-parameter-sets>`_
+10. `Download example forcing <#download-example-forcing>`__
+11. `Download observation data <#download-observation-data>`__
+
+Conda environment
+-----------------
+
+The ewatercycle Python package uses a lot of geospatial dependencies
+which can be installed easiest using `Conda <https://conda.io/>`__ package
+management system.
+
+Instal Conda by using the `miniconda
+installer <https://docs.conda.io/en/latest/miniconda.html>`__. For more thourough instuctions see `HPC to cluster <hpc_to_cluster.rst>`_ chapter.
+
+After conda is installed you can install the software dependencies with
+a `conda environment
+file <https://github.com/eWaterCycle/ewatercycle/blob/master/environment.yml>`__.
+
+.. code:: shell
+
+    wget https://raw.githubusercontent.com/eWaterCycle/ewatercycle/master/environment.yml
+
+.. code:: shell
+
+    conda env create --file environment.yml
+
+.. code:: shell
+
+    conda activate ewatercycle
+
+Do not forget that any terminal or Jupyter kernel should activate the conda environment before the ewatercycle Python package can be used.
+
+Install ewatercycle package
+---------------------------
+
+The Python package can be installed using pip
+
+.. code:: shell
+
+    # TODO once released install from PyPi instead of GitHub
+    # pip install ewatercycle
+    pip install git+https://github.com/eWaterCycle/ewatercycle.git#egg=ewatercycle
+
+
+Configure ESMValTool
+--------------------
+
+ESMValTool is used to generate forcing (temperature, precipitation,
+etc.) files from climate data sets for hydrological models. The
+ESMValTool has been installed as a dependency of the package.
+
+See https://docs.esmvaltool.org/en/latest/quickstart/configuration.html
+how configure ESMValTool
+
+Download climate data
+---------------------
+
+The ERA5 and ERA-Interim climate datasets can be used to generate
+forcings
+
+ERA5
+~~~~
+
+To download ERA5 data files you can use the
+`era5cli <https://era5cli.readthedocs.io/>`__ tool.
+
+.. code:: shell
+
+    pip install era5cli
+
+To run the example notebooks the hourly ERA5 data files for year 1990
+and 1991 and for variables: pr, psl, tas, taxmin, tasmax, tdps, uas,
+vas, rsds, rsdt and fx orog need to be downloaded with
+
+.. code:: shell
+
+    # TODO is era5cli command is correct?
+    cd <ESMValTool raw directory>
+    era5cli hourly --startyear 1990 --endyear 1991 --variables pr psl tas taxmin tasmax tdps uas vas rsds rsdt orog
+    cd -
+
+.. code:: shell
+
+    # TODO cmorize files?
+
+ERA-Interim
+~~~~~~~~~~~
+
+ERA-Interim has been superseeded by ERA5, but could be useful for
+reproduction studies and its smaller size. The ERA-Interim data files
+can be downloaded at
+https://www.ecmwf.int/en/forecasts/datasets/reanalysis-datasets/era-interim
+
+Installing a container engine requires root permission on the machine.
```

Comment:
This line should be moved to the next section.

Reply:
done